packages/muelu/src/Graph/Containers/MueLu_Aggregates_def.hpp

```diff
@@ -130,16 +132,20 @@ namespace MueLu {
   std::string Aggregates<LocalOrdinal, GlobalOrdinal, Node>::description() const {
     std::ostringstream out;
     out << BaseClass::description();
-    out << "{nGlobalAggregates = " << GetNumGlobalAggregates() << "}";
+    if (nGlobalAggregates_ == -1) out << "{nGlobalAggregates = not computed}";
```

Comment:
@jhux2  I'm a little confused by your comment. Could you extremely explicit? What should be changed?

Reply:
@jhuJust to be a little clear on the context. I got rid of the function GetNumGlobalAggregates() and replaced it by GetNumGlobalAggregatesComputeIfNeeded().  The new function is now public. I kind of want it so that if describe() is called and the aggregates are not computed ... that this is printed out ... as opposed to invoking the new function which will then invoke communication to try and compute the number of aggregates.

Reply:
What I meant was that on line 136 the print does
```
out << "{nGlobalAggregates = " << nGlobalAggregates_ << "}";
```
Compare that to
https://github.com/trilinos/Trilinos/blob/ae15c55861761631674ea9a33a5028c1ee6e1eec/packages/muelu/src/Graph/Containers/MueLu_Aggregates_def.hpp#L147

Reply:
@jhux2 So you mean ... that one prints the curly braces and the other doesn't? Is that it?

Reply:
Ah, the first print with the curly braces is in the `describe` method.  The second print is in the aggregate `print` method.

These serve two different purposes, so please ignore my comment.